cime_config/testdefs/testlist_clm.xml

```diff
@@ -1870,6 +1870,15 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
```

Comment:
@ekluzek , I like this test.  What is its origin?

Reply:
You mean the Hydro test I added right? SMS_Lm3_D_Mmpi-serial.1x1_brazil.I2000Clm50FatesCruGs.clm-FatesHydro. I wanted a test to verify that FatesHydro worked, even though the ERS test for it fails. Since, this passes on cheyenne, and only MEM_LEAK fails on hobart_nag it shows that FatesHydro does work. This would help catch a problem if FatesHydro gets screwed up.